doc/plugin-develop.md

```diff
@@ -295,7 +295,188 @@ end
 
 ## implement the logic
 
-Write the logic of the plugin in the corresponding phase.
+Write the logic of the plugin in the corresponding phase. There are two parameters `conf` and `ctx` in the `phase` method, take the `limit-conn` plugin configuration as an example.
```

Comment:
Better to use "phase method" instead of "`phase` method" as there is not a method called `phase`.

Reply:
updated.